Photoview/Views/MediaDetails/ThumbnailDetailsView.swift

```diff
@@ -116,10 +116,10 @@ extension ThumbnailDetailsView {
                             previousOffset = currentOffset
                         })
                 )
-                //.onTapGesture {
+                .onTapGesture {
                     // Disabled until a better solution is made
-                    // mediaEnv.fullScreen = !fullscreenMode
-                //}
+                     mediaEnv.fullScreen = !fullscreenMode
+                }
```

Comment:
I'm curious why this was commented out in the first place why it said "until a better solution is made"?